fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -869,7 +869,7 @@ public static void loadJournal(Env env, JournalEntity journal) {
                     env.getCatalogMgr().replayRefreshCatalog(log);
                     break;
                 }
-                case OperationType.OP_MODIFY_TABLE_ADD_OR_DROP_COLUMNS: {
+                case OperationType.OP_MODIFY_TABLE_ADD_OR_DROP_OR_MODIFY_COLUMNS: {
```

Comment:
we cannot change OperationType.OP_MODIFY_TABLE_ADD_OR_DROP_COLUMNS, because the op code may write disk in previous doris version, this may cause compatibility problem

Reply:
I just changed the name, what do you think about the name OP_MODIFY_TABLE_LIGHT_SCHEMA_CHANGE?

Reply:
> I just changed the name, what do you think about the name OP_MODIFY_TABLE_LIGHT_SCHEMA_CHANGE?

1、you are right! Just changed the name will not cause compatibility problem.
 2、 OP_MODIFY_TABLE_LIGHT_SCHEMA_CHANGE is a more comfortable name, the function name 
 how about  replace 'modifyTableAddOrDropOrModifyColumns()' with 'modifyTableLighSchemaChange()'?

Reply:
Great idea, I accidentally left out